package.json

```diff
@@ -6,16 +6,20 @@
   "type": "module",
   "scripts": {
     "build": "npm run build -ws",
+    "link-sdk": "yalc link @rtsdk/topia",
```

Comment:
Not suggesting an improvement here but I've always wondered why we need these yalc scripts. Can you please explain @LinaBell ?

Reply:
This allows us to test changes to the SDK locally (prior to them being published)